n/njakatiana.md

```diff
@@ -0,0 +1,9 @@
+# Njakatiana
+
+![Njakatiana](njakatiana.jpg)
+
+Njakatiana in his real name Raharisoa Njakatiana is a Malagasy singer, who was born on 07th june 1968. He started his career as a singer in 1992. He is one of the best Malagasy singer.
+
+**Genre:** [Salegy](https://en.wikipedia.org/wiki/Salegy), Slow
```

Comment:
No need to mention the link to Salegy here.

Reply:
I made the changes, please check.